homework/homework.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>zaarab's HTML homework</title>
+<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+<div align="center" class="divClass">
+	<h1>Memunat Manzuma</h1>
+    <h3>Paritie Precousre</h3>
+    <h4>HTML/CSS homework</h4>
+</div>
+<div align="center" class="divClass">
+	<span id="spanId">My favorite food is Jollof Rice</span>
+</div>
+<div align="center" id="thirdDiv">
+	<ul>
+    	<li> <a href="http://lorempixel.com/400/200/food/"><img src="jollof-rice1.jpg" title="Jollof Rice"></a></li>
+        <li><a href="http://lorempixel.com/400/200/food/"> <img src="jollof-rice2.jpeg" title="Jollof Rice and Egg"></a></li>
```

Comment:
review the naming convention of your image file to describe the content/type of the image